app/models/main.py

```diff
@@ -1351,6 +1363,8 @@ def status(cls):
         "live": 0,
         "closed": 1,
         "awarded": 1,
+        "cancelled": 1,
+        "unsuccessful": 1,
```

Comment:
What does this ordering do? I'm just curious as I've not used it before.

Reply:
On the opportunity listings, the Briefs were previously sorted by status/publish date/ID, where 'status' was descending alphabetically (because there were just `live` and `closed` in the list). Now we want `awarded`, `cancelled` and `unsuccessful` to have the same sort level as `closed` Briefs, so we have this mapping.

Reply:
Nice.